management/client.go

```diff
@@ -54,6 +54,21 @@ func NewClientFromConfig(subscriptionID string, managementCert []byte, config Cl
 	return makeClient(subscriptionID, managementCert, config.ManagementURL)
 }
 
+// NewClientFromPublishSettingsFile creates a new Client using a publishsettings file
+func NewClientFromPublishSettingsFile(filePath string) (Client, error) {
```

Comment:
We need another constructor that take publish settings file and config (like NewClientFromConfig)

How about these two:
- NewClientFromPublishSettings
- NewClientFromPublishSettingsAndConfig

see the existing constructors above.